src/app/js/formats/shared/LodexResource.js

```diff
@@ -39,7 +39,28 @@ const styles = stylesToClassname(
 );
 
 // see https://jsonfeed.org/version/1#items
-const LodexResource = ({ id, url, title, summary }) => {
+const LodexResource = props => {
+    let id = props.id;
+    let url = props.url;
+    let title = props.title;
+    let summary = props.summary;
+    let summarySize = props.summarySize;
+    let openInNewTab = props.openInNewTab;
+
+    if (summarySize > 0) {
+        let currentSize = 0;
+        let finalOutput = '';
+        for (const substr of summary.split(' ')) {
```

Comment:
You should put summary.split in a "const" before the "for" to avoid spliting at each iteration.

Reply:
with a forEeach call after the split this can work also ?
```js
summary.split(' ').forEach(e => {/* code */})
```